repology/parser/rubygem.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2017 Steve Wills <steve@mouf.net>
+#
+# This file is part of repology
+#
+# repology is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# repology is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with repology.  If not, see <http://www.gnu.org/licenses/>.
+
+import types
+from rubymarshal.reader import loads, load
```

Comment:
I'd prefer ```import rubymarshal.reader``` and using fully-qualified names, otherwise it's not clear what ```load``` actually does.